src/Feature/Account/client/Integration/Account/reducer.ts

```diff
@@ -23,7 +23,7 @@ export const initialSignUpState: SignUpState = {
   accountValidation: {
     email: '',
     inUse: false,
-    invalid: true,
+    invalid: false,
```

Comment:
Why this value changed to `false`? (Is form submit button became clickable with clear form on start?)

Reply:
because email don't invalid by default